### PR TITLE
[#81294712] categories menu position for tablet and mobile

### PR DESCRIPTION
**Pull request by jaime-evr**
## Description

- Fix position of categories menu.
- Smaller crowd logo image on mobile.

### CODE REVIEW

**Comment by berkowitz** [comment]
LGTM :shipit:

File: app/views/application/_we_share.html.erb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <section id="share">
   <h3 class="share-title is-not-mobile">We share what we know</h3>
+  <div class="categories index categories-mobile">
+    <%= select_tag :category, options_from_collection_for_select(@categories, 'id', 'name'), class: 'categories-select' %>
```
Comment (heridev) [inline app/views/application/_we_share.html.erb]: @jaime-evr I always try to write lines with 80 characters or less in this case I would prefer to format this line as follow:
```
<%= select_tag :category,
                          options_from_collection_for_select(@categories, 'id', 'name'),
                          class: 'categories-select' %>
```

**Comment by heridev** [comment]
in spite of my comment LGTM

**Comment by heridev** [comment]
long lines

**Comment by jaime-evr** [comment]
jaja, thanks @heridev, done :+1:

**Comment by heridev** [comment]
terrific! haha

**Pull request merged**